src/wp-includes/rest-api/endpoints/class-wp-rest-users-controller.php

```diff
@@ -1608,6 +1629,16 @@ public function get_collection_params() {
 			),
 		);
 
+		$query_params['search_columns'] = array(
```

Comment:
I think we should add the `@since 6.8.0 Added `search_columns` property.`, no?

Reply:
ok